ReactWindows/ReactNative.Shared/UIManager/BorderedCanvas.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
 
 #if WINDOWS_UWP
```

Comment:
># [](start = 0, length = 1)

Which React Native components are you intending to make accessible with this change? (e.g. View, ScrollView)

Reply:
It's for View (or whatever component that finally renders a view with the accessibility props propagated to it)

Reply:
Button and View as I tersely mentioned in title of PR. Will update to less terse version in next iteration.

---
In reply to: [158561839](https://github.com/reseul/react-native-windows/pull/1#discussion_r158561839) [](ancestors = 158561839)